README.md

```diff
@@ -52,37 +51,42 @@ and [generate an API token](https://docs.databricks.com/dev-tools/api/latest/tok
 For security reasons, we recommend using a Databricks service principal AAD token.
 
 ### Create an Azure Service Principal
-You can:
+Here are two ways that you can create an Azure Service Princiapl. The first way is via the UI. You follow do so in the Azure Portal, see the [Azure Databricks documentation](https://docs.microsoft.com/en-us/azure/databricks/administration-guide/users-groups/service-principals#create-a-service-principal). Record the Application (client) Id, Directory (tenant) Id, and client secret values generated by the steps.
```

Comment:
`You follow do so`, please reword this

Reply:
Also might be good to actually have the varaible names: e.g.  Application Id(`appId`)

Reply:
There are no variable names in the UI process, thus I have no included them.